css/style.css

```diff
@@ -0,0 +1,454 @@
+}
+
+.main-menu .nav-header {
+  float: left;
+  margin-left: 60px;
+}
+.main-menu .nav-header li {
+  float: left;
+}
+.main-menu .nav-header a {
+  display: block;
+  line-height: 30px;
+  color: #83909d;
+  margin: 0 24px;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  position: relative;
+}
+
+.main-menu .nav-right-header {
+  float: right;
+  margin-left: 50px;
+}
+.main-menu .nav-right-header li {
+  line-height: 30px;
+  float: left;
+  display: block;
+  margin: 0 20px;
+}
+
+.main-menu a:hover {
+  color: #00bcd4;
+  cursor: pointer;
+}
+.top-nav a.icon {
+  display: none;
+}
+.main-menu .nav-header .active {
+  margin-top: -5px;
+  font-weight: bold;
+  color: #ffffff;
+  border: 2px solid #ffffff;
+}
+.main-menu .nav-header .icon {
+  height: 18px;
+}
+
+/* ----------------slider-dark---------------- */
+.slider-dark {
+  color: #ffffff;
+  max-height: 900px;
+  text-align: center;
+  padding-top: 60px;
+}
+.slider-dark img,
+.slider img {
+  position: relative;
+  margin: 120px auto 0px;
+}
+.slider-dark h1 {
+  font-family: "Montserrat - Ultra Light";
+  font-size: 70px;
+  font-weight: 400;
+  line-height: 50px;
+  margin: 110px auto 70px;
+  text-transform: uppercase;
+}
+.slider-dark span {
+  text-transform: uppercase;
+  font-weight: 100 !important;
+}
+.slider-dark p {
+  margin: 65px auto 0;
+  line-height: 25px;
+  margin-left: 60px;
+}
+.slider-dark .btn {
+  display: inline-block;
+  border-radius: 50px;
+  font-weight: 500;
+  padding: 10px 20px;
+  margin: 80px 25px 150px;
+  cursor: pointer;
+}
+.slider-dark .purchase {
+  color: #ffffff;
+  border: 2px solid #ffffff;
+  background-color: #2c2e3d;
+}
+.slider-dark .more-detail {
+  color: #00bcd4;
+  border: 2px solid #00bcd4;
+  background-color: #2c2e3d;
+}
+.slider-dark li {
+  position: relative;
+  display: inline;
+  color: #ffffff;
+  margin: 23px;
+}
+.slider-dark .round-pager .fa-circle {
+  color: #00bcd4;
+  border: 1px solid #ffffff;
+  border-radius: 50%;
+}
+
+/* ---------arrow---------- */
+.slider-dark a {
+  color: #ffffff;
+  z-index: 9;
+  position: relative;
+}
+.fa-arrow-left {
+  float: left;
+  background-color: #212331;
+  padding: 20px;
+  width: 80px;
+  transform: translateX(-120px) translateY(-450px);
+}
+.fa-arrow-right {
+  float: right;
+  background-color: #212331;
+  padding: 20px;
+  width: 80px;
+  transform: translateX(120px) translateY(-450px);
+}
+.fa-arrow-left:hover,
+.fa-arrow-right:hover {
+  background-color: #00bcd4;
+}
+
+/* ------------- title-light ----------- */
+.title-light {
+  max-height: 590px;
+}
+.title-light h2 {
+  text-align: center;
+  font-size: 40px;
+  font-weight: 400;
+  margin: 140px auto 77px;
+  font-family: "Myriad Pro";
+}
+.title-light span {
+  font-weight: bold;
+}
+.title-light p {
+  clear: left;
+  font-size: 16pt;
+  line-height: 32px;
+  margin-bottom: 160px;
+  font-family: "Source Sans Pro";
+}
+
+/* --------------tables--------------------- */
+.tables {
```

Comment:
Như chị nói ở trên em dùng chung 1 class cho toàn bộ các khối có kích thước chung và ko dùng margin như này nhé